source/class/qx/tool/cli/commands/Serve.js

```diff
@@ -34,105 +34,38 @@ qx.Class.define("qx.tool.cli.commands.Serve", {
 
   statics: {
 
+    builder: {
```

Comment:
I think there's a style guide somewhere that says that static values are supposed to have upper case names?  Also it would be good to see this named to show that it's for yargs, eg `YARGS_BUILDER` rather than just `builder`

Reply:
I'll fix this in #462 , on which this PR is based.

Reply:
Fixed in https://github.com/qooxdoo/qooxdoo-compiler/pull/462/commits/5b5a53d4efa05dbefb35f07532b28a9aec938614 , please approve & merge that PR first.